src/Processor/AbstractDisputeNotificationProcessor.php

```diff
@@ -0,0 +1,65 @@
+<?php declare(strict_types=1);
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Webhook Module for PHP
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Webhook\Processor;
+
+use Adyen\Webhook\PaymentStates;
+
+
+
+abstract class AbstractDisputeNotificationProcessor extends Processor implements ProcessorInterface
+{
+    const DISPUTE_STATUS_UNDEFENDED = "Undefended";
+    const DISPUTE_STATUS_LOST = "Lost";
+    const DISPUTE_STATUS_ACCEPTED = "Accepted";
+    const FINAL_DISPUTE_STATUSES = [
+        self::DISPUTE_STATUS_LOST,
+        self::DISPUTE_STATUS_ACCEPTED,
+        self::DISPUTE_STATUS_UNDEFENDED
+    ];
+    const DISPUTE_STATUS = "disputeStatus";
+
+
+    const ORDER_ARRAY = [
```

Comment:
Can you please rename it to more self-explanatory name? `CHARGEBACK_ORDER_STATES` maybe